src/zero/api/handlers.clj

```diff
@@ -125,6 +136,7 @@
 (defn post-start
   "Start the workloads with UUIDs in REQUEST."
   [request]
+  (log/info "post-start endpoint parameters:" (:parameters request))
```

Comment:
We might want to double check there are no credentials in the parameters, for example, auth tokens.

Reply:
I think I may have resolved this, would love another quick pass through to see if more narrowly logging is sufficient